test/Azure.Functions.Cli.Tests/ActionsTests/ResolveActionTests.cs

```diff
@@ -93,7 +93,8 @@ private IContainer InitializeContainerForTests()
                 .SingleInstance();
 
             builder.RegisterType<ProcessManager>()
-                .As<IProcessManager>();
+                .As<IProcessManager>()
```

Comment:
Reviewed usage and this is not currently used by anything outside of the new surface used for .NET 8.